app/models/post.rb

```diff
@@ -1,11 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- Hey, kindly leave this as the author which references the user's table to maintain the ERD diagram as you've changed it currently and this doesn't follow the ERD diagram based on the project requirement.